contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -59,6 +60,12 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, Reentrancy
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     IDisputeModule public immutable DISPUTE_MODULE;
 
+    modifier whenNotPaused() {
```

Comment:
why do you need to add the modifier here? not other places?

Reply:
Because `IpRoyaltyVault` does not inherit from OZ's `Pausable`. The vaults don't have their own `paused` state, they check `RoyaltyPolicyLAP`s.
It is not feasible to pause all vaults 1 by 1 or even batching tx.